apps/settings/style/icons.css

```diff
@@ -295,6 +295,7 @@ aside.connecting {
 html[dir="rtl"] a.menu-item:after {
   left: 0.3rem;
   right: unset;
+  content: "back-light";
```

Comment:
I would suggest to make the change when we decide to change the direction of the transition.